### PR TITLE
Add org.kde.francis

### PR DESCRIPTION
**Pull request by CarlSchwan**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

**Comment by CarlSchwan** [comment]
@flathub/kde :)

**Comment by Justinzobel** [comment]
Looks good :+1:

File: org.kde.francis.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DBUILD_TESTING=OFF",
+                "-DCMAKE_BUILD_TYPE=Release"
+            ],
+            "builddir": true,
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/francis/francis-1.0.1.tar.xz",
```
Comment (bbhtt) [inline org.kde.francis.json:48]: The first and last one needs to be fixed.

```
appstream-util validate org.kde.francis.appdata.xml 
org.kde.francis.appdata.xml: FAILED:
• tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
• style-invalid         : <image> has vertical padding [https://cdn.kde.org/screenshots/francis/main.png]
• style-invalid         : <image> has horizontal padding [https://cdn.kde.org/screenshots/francis/main.png]
• style-invalid         : <caption> is too short [Main View];shortest allowed is 10 chars
Validation of files failed
```

File: org.kde.francis.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    "sdk": "org.kde.Sdk",
+    "command": "francis",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri"
+    ],
+    "separate-locales": false,
+
```
Comment (bbhtt) [inline org.kde.francis.json:14]: Add cleanup `/lib/cmake`

File: org.kde.francis.json (hunk 1 above)
Comment (bbhtt) [reply org.kde.francis.json:48]:
```
desktop-file-validate org.kde.francis.desktop 
org.kde.francis.desktop: error: file contains key "SingleMainWindow" in group "Desktop Entry", but keys extending the format should start with "X-"
```

desktop-file-validate from Freedesktop SDK is missing this. This is not included in a desktop-file-utils release, distributions patch it

**Comment by hfiguiere** [comment]
bot, build org.kde.francis

**Comment by bbhtt** [comment]
About page seems to be empty:

```
QQmlComponent: Component is not ready
file:///usr/lib/qml/org/kde/kirigami.2/PageRow.qml:914: Error: Error while loading page: qrc:/About.qml:8 module "org.kde.kirigamiaddons.formcard" is not installed
```

<details>
<summary> About </summary>

image

</details>

File: org.kde.francis.json (hunk 1 above)
Comment (bbhtt) [reply org.kde.francis.json:48]: > appstream runs desktop file validation, so it'll fail I think.

Looks like it got reduced to a warning, so it should be fine

```
WARNING: Failed to validate desktop file /srv/buildbot/worker/build-x86_64-3/build/builddir/export/share/applications/org.kde.francis.desktop: /srv/buildbot/worker/build-x86_64-3/build/builddir/export/share/applications/org.kde.francis.desktop: error: file contains key "SingleMainWindow" in group "Desktop Entry", but keys extending the format should start with "X-"
```

**Comment by CarlSchwan** [comment]
> About page seems to be empty:
> 
> ```
> QQmlComponent: Component is not ready
> file:///usr/lib/qml/org/kde/kirigami.2/PageRow.qml:914: Error: Error while loading page: qrc:/About.qml:8 module "org.kde.kirigamiaddons.formcard" is not installed
> ```
> 
> About

fixed by using kirigami addons 0.11.0 instead of 0.10.

File: org.kde.francis.json (hunk 1 above)
Comment (CarlSchwan) [reply org.kde.francis.json:50]: > • style-invalid         : <image> has vertical padding [https://cdn.kde.org/screenshots/francis/main.png]
> • style-invalid         : <image> has horizontal padding [https://cdn.kde.org/screenshots/francis/main.png]

Looks like the checker doesn't like the screenshot taken with the official guidelines for taking screenshot in KDE :( Fortunately this is just a warning

**Comment by CarlSchwan** [comment]
bot, build org.kde.francis

File: org.kde.francis.json (hunk 1 above)
Comment (hfiguiere) [reply org.kde.francis.json:50]: The check that buildbot use doesn't even emit a warning for this.

File: remove-singlewindow.patch
Comment (bbhtt) [inline remove-singlewindow.patch:1]: This seems empty? You don't need to add this, since it is a warning from appstream. not an error as I suspected.

**Comment by bbhtt** [comment]
bot, build org.kde.francis

**Pull request closed without merging**